src/ert/cli/main.py

```diff
@@ -44,6 +44,15 @@ def run_cli(args):
     os.chdir(res_config.config_path)
     ert = EnKFMain(res_config)
     facade = LibresFacade(ert)
+    if not facade.have_observations and args.mode not in [
+        ENSEMBLE_EXPERIMENT_MODE,
+        TEST_RUN_MODE,
```

Comment:
What about workflow mode, not that I have ever used it?

Reply:
Good point `WORKFLOW_MODE` seems to be something different might not require observations.